.vscode/launch.json

```diff
@@ -0,0 +1,36 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
If VSCodes doens’t care then I don’t care but comments aren’t valid in JSON.

Reply:
vscode does not care 🙂